template.yml

```diff
@@ -0,0 +1,101 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform:
+  - AWS::LanguageExtensions
+  - AWS::Serverless-2016-10-31
+
+Description: >
+  Create cloud formation stack for restapi, ip authorizer lambda, backend lambda
+
+Globals:
+  Function:
+    Timeout: 30
+    MemorySize: 256
+
+Parameters:
+  IP_RANGE:
```

Comment:
If we're going to use parameter store to pull these, these parameters need to be the **names** of the parameter store parameters.